discord/ui/select.py

```diff
@@ -327,8 +665,14 @@ def select(
     In order to get the selected items that the user has chosen within the callback
     use :attr:`Select.values`.
 
+    .. versionchanged:: 2.3
+
+        Creating other select menus is now supported.
```

Comment:
This could be clearer:
```suggestion
        Creating select menus of different types is now supported.
```